src/ui/public/index_patterns/index_patterns.js

```diff
@@ -41,6 +41,7 @@ export function IndexPatternsProvider(Notifier, Private) {
   self.cache = patternCache;
   self.getIds = getProvider('id');
   self.getTitles = getProvider('attributes.title');
+  self.getIndexPatterns = getProvider();
```

Comment:
Since the service is called indexPatterns, I think the names getIds and getTitles makes a lot of sense. Maybe the new method should be named getSavedObjects?